index.html

```diff
@@ -1,14 +1,284 @@
-  </head>
-  <body>
-    <section id="header"></section>
-    <section id="features"></section>
-    <section id="main"></section>
-    <section id="footer"></section>
-  </body>
+
+<!-- Basic Page Setup -->
+
+<head>
+  <title>Mockup Home</title>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
+  <link href='/mockUpCss/index.css' rel='stylesheet'>
+</head>
+
+<!-- Full Layout -->
+
+<body>
+
+  <!-- Top Section -->
+  <section id="header" class="centered-titles">
+    <h1><a href="/index.html">
+        HOME PAGE
+      </a></h1>
+    <h6>
+      VILLAINS ANONYMOUS
+    </h6>
+  </section>
+
+  <!-- Top Features -->
+  <h5 class="centered-titles">
+    WINNERS OF VILLAINS ANONYMOUS AWARDS
+  </h5>
+
+  <!-- Feature Pictures -->
+  <section id="image-box">
+    <img src='/images/sauron.webp' class="flex-pictures" id="flex-pic-1">
+    <img src='/images/vader.jpg' class="flex-pictures">
+    <img src='/images/smith.jpg' class="flex-pictures" id="flex-pic-3">
+  </section>
+
+  <!-- Feature Title -->
+  <section id="image-titles">
+    <div>
+      MOST POWERFUL
+    </div>
+    <div>
+      OVERLORD OF THE CENTURY
+    </div>
+    <div>
+      BEST IN TECH
+    </div>
+  </section>
+
+  <!-- Music Button -->
+  <div align="center" id="music-button">
+    <audio src="music/Star Wars- The Imperial March (Darth Vader's Theme).mp3" controls>
+    </audio>
+  </div>
+
+  <!-- Feature Link Button -->
+  <div class="centered-titles" id="top-button">
+    <a href="learn-more.html"><button id="top-button-more">
+        TELL ME MORE
+      </button></a>
+  </div>
+
+  <!-- Large Quote Block -->
+  <h5 class="centered-titles" id="quote-block">
+    "EVERY VILLIAN IS A HERO IN THEIR OWN MIND"
+  </h5>
+
+  <!-- Mid-Page Flex-Box -->
+  <section id="main-box">
+
+    <!-- Left Column -->
+    <section id="main-left">
+      <h5>
+        WHY MAGIC ALONE JUST DOESN'T CUT IT
+      </h5>
+      <div>
+        <img src='/images/maleficent.jpg'>
+      </div>
+
+      <!-- Main Article -->
+      <h5>
+        AND IS MALEFICENT STILL A VILLIAN?
+      </h5>
+      <p>
+        Lorem ipsum dolor sit amet consectetur, adipisicing elit. Nesciunt accusamus quibusdam veritatis non facilis
+        repudiandae saepe, voluptates aspernatur illo animi tempore minima adipisci! Omnis a nostrum eaque. Neque
+        officiis
+        nihil quasi rem modi. Quibusdam magni laudantium laboriosam libero, ratione ea facere amet aperiam totam
+        voluptatibus accusamus hic quaerat ipsa dolor!
+        Lorem, ipsum dolor sit amet consectetur adipisicing elit. Odio, deleniti blanditiis. Quae quisquam, dolor
+        necessitatibus enim delectus nulla ea dolorem beatae mollitia harum quos assumenda ipsum. Officia dicta culpa
+        minus.`
+      </p>
+
+      <!-- Button Link -->
+      <a href="/continue-reading.html"><button id="large-button">
+          CONTINUE READING
+        </button></a>

```

Comment:
The buttons from the sample contained images inside of them. If you want to go over how to make that happen we can do that during a tutor session or during some free time in class.

Reply:
This comment also pertains to every button through your project